test/geometry/transform/test_affine.py

```diff
@@ -28,6 +28,11 @@ def test_downsize(self, device, dtype):
         out = kornia.resize(inp, (3, 1))
         assert out.shape == (1, 3, 3, 1)
 
+    def test_downsizeAA(self, device, dtype):
+        inp = torch.rand(1, 3, 10, 8, device=device, dtype=dtype)
+        out = kornia.resize(inp, (5, 3), antialias=True)
+        assert out.shape == (1, 3, 5, 3)
```

Comment:
add some value check?

Reply:
Apparently our resize as a whole do not have any value check :) But yes, good point